### DISCUSSION TITLE
posts/django%E6%95%99%E7%A8%8B/

### DISCUSSION THREAD

**Discussion by giscus** [roninro/hugo-theme-puppet#67] [General]
# posts/django%E6%95%99%E7%A8%8B/

Django 教程[toc] http 4 种请求和传参方式网站开发逻辑梳理初级阶段，使用 admin 后台快速建站 model admin 直接编写后台逻辑和前台模板页面 url view FBV CBV template form view 之上的逻辑中

https://zhenda-hub.github.io/posts/django%E6%95%99%E7%A8%8B/

**Comment by zhenda-hub** [comment]
good
